ui/layouts/header.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { useLocation } from 'react-router-dom';
+import { AppHeader } from '../components/multichain';
+import { hideAppHeader, showAppHeader } from '../pages/routes/utils';
+
+export const Header = () => {
+  const location = useLocation();
+
+  return (
+    <>
+      {process.env.REMOVE_GNS
+        ? showAppHeader({ location }) && <AppHeader location={location} />
+        : !hideAppHeader({ location }) && <AppHeader location={location} />}
```

Comment:
I know this is not your doing but `showAppHeader` and `hideAppHeader` not being opposites of eachother is really confusing 😅

Reply:
@ameliejyc IKR! 

For a next PR since the logic of hiding it shouldn't be in that function and will be taken care of upcoming layouts moving forward